src/openrct2-ui/windows/TitleMenu.cpp

```diff
@@ -113,7 +118,9 @@ rct_window* window_title_menu_open()
         i++;
     }
     window->width = x;
+    window->widgets[WIDX_NEW_VERSION].right = window->width;
     window->windowPos.x = (context_get_width() - window->width) / 2;
+    window->colours[1] = TRANSLUCENT(COLOUR_LIGHT_ORANGE);
```

Comment:
We don't usually hardcode our window colours. This should be done using a theme class. Orange is a striking default, though — good choice.